cmd/recording.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+  "github.com/applandinc/appland-cli/internal/recording"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	var (
+		recordingCmd = &cobra.Command{
+			Use:   "recording",
+			Short: "Manage AppMap recordings",
+			Run: func(cmd *cobra.Command, args []string) {
+				cmd.Usage()
+			},
+		}
+		recordingStartCmd = &cobra.Command{
+			Use:   "start [url]",
+			Short: "Start a new AppMap recording session",
+			Args:  cobra.ExactArgs(1),
+      Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Always use `gofmt`, which will be applied automatically if you use a tool like VSCode.